src/exporters/vhi.py

```diff
@@ -196,25 +167,38 @@ def check_failures(self) -> List:
 
         with open(pickled_error_fname, 'rb') as f:
             errors = pickle.load(f)
-        print("*************************")
-        print("Errors:")
-        pprint(errors)
 
         return errors
 
-    def export(self) -> List:
-        """Export VHI data from the ftp server
+    @staticmethod
+    def get_default_years():
+        """ returns the default arguments for no. years """
+        years = [yr for yr in range(1981, 2020)]
+
+        return years
+
+    def export(self, years: List = [yr for yr in range(1981, 2020)]) -> List:
+        """Export VHI data from the ftp server.
+        By default write output to raw/vhi/{YEAR}/{filename}
 
-        by default write output to raw/vhi/{YEAR}/{filename}
-        self.data_folder
-        self.raw_folder
+        Arguments:
+        ---------
+        years : List
+            list of years that you want to download. Default `range(1981,2020)`
+
```

Comment:
@gabrieltseng is the docstring okay?

https://github.com/esowc/ml_drought/pull/12#discussion_r285193949

Reply:
Yep! This is great, thanks :)

Reply:
yes thank you!